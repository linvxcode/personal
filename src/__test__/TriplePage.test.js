
import 'intersection-observer'
import React from "react";
import { render, fireEvent,screen} from "@testing-library/react";
import TriplePage from "../Trple Page/TriplePage";

describe('triple', () => {
    it('Render', () => {
        const {triple} = render(<TriplePage />);
        // const hover = screen.getAllByTestId('hovertest');

        // fireEvent.mouseEnter(hover);

        expect(triple).toMatchSnapshot();
    })
})