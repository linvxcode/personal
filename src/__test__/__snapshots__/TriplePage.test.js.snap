// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`triple Render 1`] = `undefined`;
